src/dsp_tools/commands/excel2json/models/input_error.py

```diff
@@ -240,3 +241,44 @@ def execute_error_protocol(self) -> str:
         if self.message_path:
             msg.append(f"The error occurred at {self.message_path}")
         return separator.join(msg)
+
+
+@dataclass(frozen=True)
+class ListNodeProblem:
+    node_id: str
+    problems: dict[str, str]
+
+    def execute_error_protocol(self) -> str:
+        msg = [
+            f"The node '{self.node_id}' has the following problem(s):",
+        ]
+        msg.extend([f"Field: '{key}' Problem: {value}" for key, value in self.problems.items()])
```

Comment:
Not sure if I got this correctly, but what do you mean with "Field"? Is it the Excel column name? Then I would call it like that.

Reply:
Well it is not quite the excel column name, for example the labels are in several columns, so I can't call it column...